pkg/apis/etcd/v1beta2/backup_types.go

```diff
@@ -79,6 +79,10 @@ type BackupCRStatus struct {
 	Succeeded bool `json:"succeeded"`
 	// Reason indicates the reason for any backup related failures.
 	Reason string `json:"Reason,omitempty"`
+	// EtcdVersion is the version of the backup etcd server.
+	EtcdVersion string `json:"etcdversion,omitempty"`
```

Comment:
json tag should be etcdVersion? etcdRevision?

Reply:
sounds good.

Reply:
fixed cc/ @xiang90